frontend/public/components/masthead-toolbar.jsx

```diff
@@ -6,9 +6,21 @@ import { Button, Dropdown, DropdownToggle, DropdownSeparator, DropdownItem, Keba
 
 import { FLAGS, stateToProps as flagStateToProps, flagPending } from '../features';
 import { authSvc } from '../module/auth';
-import { history } from './utils';
+import { history, Firehose } from './utils';
 import { openshiftHelpBase } from './utils/documentation';
 import { AboutModal } from './about-modal';
+import { getAvailableClusterUpdates, clusterVersionReference } from './cluster-settings/cluster-settings';
+
+const UpdatesAvailableButton = ({obj, onClick}) => {
+  const updatesAvailable = !_.isEmpty(getAvailableClusterUpdates(obj.data));
+  return updatesAvailable
```

Comment:
I'm trying to remember if instead of `: null;` you can just `return updatesAvailable && <Toolbar>....`

Reply:
Has to exclusively return null or else you get a runtime error

Reply:
Ah, so if it's not explicit, it fails.  That's unfortunate.